ucm2/alcatel-idol347/HiFi.conf

```diff
@@ -0,0 +1,101 @@
+# Use case configuration for Alcatel OneTouch Idol 3 (4.7").
+# The two speakers/earpieces are connected to external codec (TFA9897)
+# Headphones support requires writing a driver for AK4375
+
+Define {
+	WcdPlaybackPCM "hw:${CardId},1"
+	WcdCapturePCM "hw:${CardId},0"
+}
+<platforms/msm8916/qdsp6-components.conf>
+
+SectionDevice."SpeakerStereo" {
```

Comment:
I think we should at least have configurations for the standard device names, i.e.

- `Speaker` (= SpeakerStereo I guess)
- `Earpiece` (= EarpieceTop I guess)

because those may be referenced by other applications (e.g. the Plasma Mobile dialer).

You could consider having more optional configurations although not sure if it's worth it.